snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/literal1.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+public class Example
+{
+   public static void Main()
+   {
+      int n = 9;
+      Console.WriteLine($@"{n:##.0\%}");  
+      Console.WriteLine($@"{n:\'##\'}");      
```

Comment:
do you need these special characters here at the end? I try to remove trailing spaces if possible.